src/services/authService.ts

```diff
@@ -34,10 +34,15 @@ class AuthService {
   }
   
   signInWithAzure = async () => {
-    return supabaseClient.auth.signInWithOAuth({ provider: 'azure' })
-      .then(resp => {
-        return resp
-      })
+    return supabaseClient.auth.signInWithOAuth({
```

Comment:
Nit: Could just return the call instead of wrapping in an async function? Would be more conscise, unless you wanted to handle the resp.error here. Same with the signInWithGoogle call above.

Reply:
Do you mean like:
```
signInWithGoogle = supabaseClient.auth.signInWithOAuth({ provider: 'google' });
```
Instead of 
```
signInWithGoogle = async () => {
    return supabaseClient.auth.signInWithOAuth({ provider: 'google' })
    .then(resp => {
      return resp
    })
  };
```
?

Wouldn't that call the function on object initialization and assign it to the promise? You might need to correct my understanding of TypeScript

Reply:
You are correct, I misspoke when I said wrapping in async function. I meant wrapping the return in a block rather than:

`signInWithAzure = () => supabaseClient.auth.signInWithOAuth({ provider: 'azure' })`